iphone/Classes/TiUIWebView.m

```diff
@@ -488,6 +488,20 @@ -(void)setScrollsToTop_:(id)value
 	[[self scrollview] setScrollsToTop:scrollsToTop];
 }
 
+- (void)setUserAgent_:(id)args
+{
+    ENSURE_TYPE_OR_NIL(args, NSString);
+    NSString *userAgent = [TiUtils stringValue:args];
+    
+    if (!userAgent) {
+        userAgent = [[TiApp app] systemUserAgent];
+    }
+    
+    NSDictionary *dict = [NSDictionary dictionaryWithObjectsAndKeys:userAgent, @"UserAgent", nil];
+    [[NSUserDefaults standardUserDefaults] registerDefaults:dict];
```

Comment:
I'm still not sure if we can do that, because it might mess-up different webviews. Let's say you set the property in webview 1 and don't set it in webview 2, webview 2 will most likely have the same user-agent as webview 1, because it is referenced globally. Correct?